CODEOWNERS

```diff
@@ -14,6 +14,7 @@
 # *                                        @galak @nashif
 
 /.known-issues/                           @inakypg @nashif
+/.github/                                 @thst-nordic
```

Comment:
This will likely cause some merge conflicts from time to time, but that seems fine.